user_guide_src/source/changelogs/v4.3.0.rst

```diff
@@ -76,6 +76,7 @@ Others
 - SQLSRV now automatically drops ``DEFAULT`` constraint when using ``Forge::dropColumn()``.
 - Added :ref:`Time::toDatabase() <time-todatabase>` to get a datetime string that can be used with databases regardless of locale.
 - You can set the locale to :php:func:`route_to()` if you pass a locale value as the last parameter.
+- Improved data returned by ``BaseConnection::getForeignKeyData()``. All DBMS return the same structure and use the same naming conventions.
```

Comment:
This is a breaking change. It changed the public method return value.
We need to add it in `BREAKING` in this changelog, and need to add what to do when upgrading in upgrading guide for v4.3.0.